code/modules/jobs/job_types/roguetown/adventurer/types/warrior.dm

```diff
@@ -0,0 +1,131 @@
+//shield sword
+/datum/subclass/sfighter
```

Comment:
Warrior could probably use level 4 weapon skills to be blunt.
At present paladin and cleric are just flat better then this for the most part.